### PR TITLE
Simplistic helpers for use in scripts/tests

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Based on #172.

**Comment by gavin-norman-sociomantic** [comment]
Rebased; unblocked.

File: src/dhtproto/client/DhtClient.d [hunk 1]
```diff
@@ -720,6 +724,68 @@ public class DhtClient : IClient
...
+            conn_notifier = delegate which is called when a connection-related
+                event occurs on a neo connection. Of type:
+                void delegate ( Neo.DhtConnNotification )
+
+    ***************************************************************************/
+
+    public this ( Neo.DhtConnectionNotifier conn_notifier = null )
+    {
+        if ( !isSchedulerUsed() )
+            initScheduler(Scheduler.Configuration.init);
```
Comment (gavin-norman-sociomantic) [inline src/dhtproto/client/DhtClient.d:751]: Mihails said elsewhere that he doesn't think doing this is advisable. Waiting for his response on https://github.com/sociomantic-tsunami/ocean/pull/612#issuecomment-414235269.

Comment (nemanja-boric-sociomantic) [reply src/dhtproto/client/DhtClient.d:751]: So, is this blocked or not :-)?

File: src/dhtproto/client/mixins/NeoSupport.d [hunk 2]
```diff
@@ -605,6 +606,59 @@ template NeoSupport ( )
...
+            hash-range queried.
+
+            Params:
+                port = port of node to connect to
+
+        ***********************************************************************/
+
+        public void connect ( ushort port )
+        {
+            this.outer.neo.addNode("127.0.0.1", port);
```
Comment (nemanja-boric-sociomantic) [inline src/dhtproto/client/mixins/NeoSupport.d]: I find usefulness of this method questionable - I think it's better to always to pass address and port to `connect` method, to make it clear where the client connects to.

File: src/dhtproto/client/DhtClient.d (hunk 1 above)
Comment (gavin-norman-sociomantic) [reply src/dhtproto/client/DhtClient.d:751]: I guess it is blocked on understanding what he meant, yes. :(

File: src/dhtproto/client/mixins/NeoSupport.d (hunk 2 above)
Comment (gavin-norman-sociomantic) [reply src/dhtproto/client/mixins/NeoSupport.d]: Yeah, it's on the edge. Saves a bit of typing, but if you think it's dangerously lazy, I can remove it.

Comment (nemanja-boric-sociomantic) [reply src/dhtproto/client/mixins/NeoSupport.d]: It's not _dangerously_ lazy, but it's a bit mysterious to me if you write `connect(port)` vs `connect(address, port)`.

**Comment by gavin-norman-sociomantic** [comment]
Updated:
* No scheduler init.
* No port-only connect.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**